ccd-definition/WorkBasketResultFields.json

```diff
@@ -6,25 +6,32 @@
     "Label": "Case title",
     "DisplayOrder": 1
   },
+  {
+    "LiveFrom": 42736,
+    "CaseTypeID": "Shared_Storage_DRAFTType",
+    "CaseFieldID": "familyManCaseNumber",
+    "Label": "FamilyMan",
```

Comment:
should it not be 2 words? or if it isnt then should it be `Familyman`?

Reply:
In the story it's written as 'FamilyMan'.

Reply:
I like FamilyMan :)